Source/WebCore/html/HTMLMediaElement.cpp

```diff
@@ -827,6 +827,12 @@ void HTMLMediaElement::attributeChanged(const QualifiedName& name, const AtomStr
     case AttributeNames::webkitwirelessvideoplaybackdisabledAttr:
         mediaSession().setWirelessVideoPlaybackDisabled(newValue != nullAtom());
         return;
+#if ENABLE(MEDIA_SOURCE)
+    case AttributeNames::webkitairplayAttr:
+        if (m_mediaSource && hasWirelessPlaybackTargetDisabled())
+            m_mediaSource->openIfDeferredOpen();
+        break;
+#endif
```

Comment:
See the comment below about this strange attribute.

Reply:
I think the behaviour is kept following this change.
`isWirelessPlaybackTargetDisabled` checks that the value is set to "deny"